privacy_policy.md

```diff
@@ -0,0 +1,45 @@
+
+USF Maps Project Privacy Policy
+
+We value your privacy, so we want to be explicit about what kind of personal information is involved when you interact with USF Maps.
+
+This page is for the USF Maps project as a whole. The project as such doesn’t collect any personally identifying information from users.  However, instances of USF Maps that serve a particular region may save log and other information, some of which is personally identifiable. Each instance should have its own privacy policy, linked from its home page.
+
+USF Maps Apps
+
+Usage of the trip planner function logs information about the requested start and endpoints, date/time, requested modes, and user IP address.  As a result of the nature of the application, it may also associate the users geolocation at the time of the request.  We realize some of this data can be sensitive, and do not plan to use it in a way that unnecessarily reveals such information to third-parties.  We expect to only make use of the information for troubleshooting problems, and conducting research on the aggregate, anonymized information contained therein.
+
+We use Google Analytics so aggregate, anonymous data about how often different app features are used can be collected.  For example, we could tell that the most used feature of USF Maps apps is the trip planner, and only 1/3 of users regularly use the maps screen (these statistics are made up for example purposes).  This data helps us better understand where to focus future improvements.  We do not collect data on individual usage of the app.  For example, we don’t know how many times User X viewed their person bookmarks, or names of those bookmarks.
+
+In the “Help” section of the apps, the app allows users to send a feedback email to the regional email address – the user can also add additional recipients to this email.  The following information is automatically added to the body of the email to help us troubleshoot issues that users encounter, with the message “The below information helps us troubleshooting problems:". The user can erase this information from the email before it is sent if the user desires not to share this information with the regional USF Maps administrators or the app maintainers:
+
+    USF Maps App version – So we know what known bugs do and do not exist in your currently-installed version of the app
+    Device make and model (e.g., iPhone 7.2, Samsung Galaxy S5) – Helps us troubleshoot device-specific problems
+    OS version (e.g., Android 6.0, iOS 9.3.1) – Helps us troubleshoot operating system version-specific problems
+
+The USF Maps app currently uses the respective APIs to show the user their real-time location and location of bus stops. These Maps APIs have fairly broad data collection abilities, and this data goes back to the respective platform vendor (Google, Apple, Microsoft). The USF Maps project doesn’t receive this data however.
```

Comment:
"this data goes back to the respective platform vendor (Google, Apple, Microsoft)" 

...can be changed to:

"some data may be collected by the respective platform vendor (Mapbox, MapQuest)."